Core/GraphQL/GraphQLManager.swift

```diff
@@ -48,7 +52,7 @@ public class GraphQLDispatcher: GraphQLDispatching {
         mutation: Query,
```

Comment:
How about we also change the generic parameter naming from `Query` to `Mutation` please? Thank you.